LOCAL_DEV.md

```diff
@@ -0,0 +1,13 @@
+To run this project locally, you need DDEV (and so, Docker) and then:
+
+```bash
+ddev start
+ddev ssh
+composer install 
+npm run dev 
+php bin/console d:s:u --dump-sql --force
+php bin/console d:f:l -e dev
```

Comment:
We should run those commands from the DDEV container.

```suggestion
ddev sf d:s:u --dump-sql --force
ddev sf d:f:l -e dev
```

Reply:
ignore this. I skipped the ddev ssh.

Reply:
i miss the `ddev sf` command availability, so i change to use it, you're right